### PR TITLE
The blacklist pull request!

### PR DESCRIPTION
**Pull request by RhiannonMichelmore**
I *think* since I've included the alembic migrate file, you should only have to run `alembic upgrade head` and it will all be good.

### CODE REVIEW

**Comment by davidjrichardson** [comment]
It's a great start but really the ergonomics of the command will be off if you have to use underscores. Discord.py has a subcommand feature that karma.py leverages and would make perfect sense here. I'd also like to see a datetime added for the karma blacklist since that information may be useful in the future!

**Comment by RhiannonMichelmore** [comment]
Okay now this should be pretty much there? check again pls :)

**Comment by RhiannonMichelmore** [comment]
R u happy now there are no black (or white) flags ;)

File: models.py [hunk 1]
```diff
@@ -119,3 +119,4 @@ class Blacklist(Base):
     __tablename__ = 'blacklist'
     name = Column(String, primary_key=True, nullable=False)
     added_by = Column(Integer, ForeignKey('users.id'),nullable=False)
+    added_at = Column(DateTime)
```
Comment (davidjrichardson) [inline models.py]: I'd rather see added_at be the following: 
```python
added_at = Column(DateTime, nullable=False, default=datetime.utcnow())
```

Its more explicit when looking at the code what the default value is.

**Pull request merged**